shell/platform/fuchsia/runtime/dart/utils/BUILD.gn

```diff
@@ -94,6 +94,7 @@ executable("dart_utils_unittests") {
     ":utils",
     "$fuchsia_sdk_root/pkg:sys_inspect_cpp",
     "//flutter/testing",
+    "//third_party/dart/runtime:libdart_jit",
```

Comment:
This is needed because dart_errors.h now references isolate related symbols, which are not exposed by `//third_party/dart/runtime:dart_api` by itself. Perhaps this dep should be in tonic's public deps, but that will make things pretty complicated because we'll have to make sure to select the right one. It's easier just to fix this target.